test/python/test_qobj.py

```diff
@@ -87,3 +90,21 @@ def test_from_dict_per_class(self):
         for qobj_class, (qobj, expected_dict) in test_parameters.items():
             with self.subTest(msg=str(qobj_class)):
                 self.assertEqual(qobj, qobj_class.from_dict(expected_dict))
+
+    def test_ibmqobj_raises_when_sending_bad_qobj(self):
+        """Test IBMQobj is denied resource request access when invalid."""
```

Comment:
Please, clarify the docstring by replacing `invalid` with `invalid Qobj instance`.